polaris/polaris/integrations/quote.py

```diff
@@ -0,0 +1,57 @@
+from typing import List
+
+from polaris.models import Quote
+
+
+class GetPricesResponse:
+    asset = ("",)
+    price = ("",)
+    decimals = 0
+
+
+class GetPriceResponse:
+    price = ("",)
+    sell_amount = ("",)
+    buy_amount = ""
+
+
+class SEP38AnchorIntegration:
+    """
+    The container class for SEP38 integrations
+
+    The list of the offline assets supported by the anchor should be returned.
+
+    The returned dictionary should contains the following attributes specified at:
+    https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0038.md#get-info
+
+    If there is no Offchain assets, an empty list is expected.
+
+    """
+
+    def get_prices(
```

Comment:
We need to pass the `SEP10Token` and `Request` object to every integration function thats called within a SEP-10 authenticated request.